test/js-api/limits.any.js

```diff
@@ -140,14 +140,15 @@ testLimit("function size", 2, kJSEmbeddingMaxFunctionSize, (builder, count) => {
         const nops = count - 2;
         const array = new Array(nops);
         for (let i = 0; i < nops; i++) array[i] = kExprNop;
+        array.push(kExprEnd);
```

Comment:
I added one byte to the function, should the `count` be updated here? I don't understand why `const nops = count - 2;`, if I add a `- 1` it will turn negative and the Array will have an invalid length.